components/leaflet-maps/ClustersMap.vue

```diff
@@ -195,6 +195,35 @@
        })
 
        return optionalsWithValue
+     },
+     getJulianDate: function (date,name){
```

Comment:
I ran this function and I don't think it returned a Julian date e.g. 24/03/2020 became 13/03/2020. Is this expected?

Reply:
This is expected.

Reply:
discussed and confirmed with Ruby. Marking as resolved.